client/gutenberg/extensions/email-subscribe/index.js

```diff
@@ -0,0 +1,37 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import EmailSubscribeEdit from './edit';
+import './editor.scss';
+
+export const name = 'email-subscribe';
+
+export const settings = {
+	title: __( 'Email Subscribe', 'jetpack' ),
+	icon: 'email',
+	category: 'widgets',
```

Comment:
Could we use the `jetpack` category? Seems like this block will be a Jetpack block 😉

Reply:
Done!